app/controllers/users_controller.rb

```diff
@@ -0,0 +1,62 @@
+class UsersController < ApplicationController
+  before_action :authorized, only: [:show]
+  def new
+    @user = User.new
+  end
+
+  def create
+    user = User.create(user_params)
```

Comment:
This line should still use `@user` because it sets user in a different way than the `user` method, but then once `@user` is set, then you can use `user` since that method defaults to `@user`.